opentasks-provider/src/main/java/org/dmfs/provider/tasks/TaskProvider.java

```diff
@@ -787,6 +798,32 @@ public int deleteInTransaction(final SQLiteDatabase db, Uri uri, String selectio
 
                 break;
             }
+
+            case INSTANCE_ID:
+                // add id to selection and fall through
+                selection = updateSelection(selectId(uri), selection);
+
+            case INSTANCES:
+            {
+                // iterate over all instances that match the selection. We iterate "manually" to execute any processors before or after deletion.
+                final Cursor cursor = db.query(Tables.INSTANCE_VIEW, null, selection, selectionArgs, null, null, null, null);
+
+                try
```

Comment:
Android Studio offers try-with-resources quick fix here now. It's also mentioned in the docs, too, that it is supported:
https://developer.android.com/studio/write/java8-support.html
Shall we start using it?

Reply:
I gave it a try.